roxctl/utils/utils_test.go

```diff
@@ -0,0 +1,46 @@
+package utils
+
+import (
+	"bufio"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_wordsAndDelimeters(t *testing.T) {
+	cases := map[string][]string{
+		"\nText with \nmultiple\n lines.": {"\n", "Text", " ", "with", " ", "\n", "multiple", "\n", " ", "lines."},
+		"":                                {},
+		" ":                               {" "},
+		" \n ":                            {" ", "\n", " "},
+		" \n\t":                           {" ", "\n", "\t"},
+		"word":                            {"word"},
```

Comment:
```suggestion
		"word":                            {"word"},
		" Leading space":                  {" ", "Leading", " ", "space"},
		"  Multiple Leading spaces":       {" ", " ", "Multiple", " ", "Leading", " ", "spaces"},
```
From testing locally, works fine 👍 . And I guess we don't need to care about other control characters like `\v` or `\r` with how we format.